src/helpers/validators/profile.js

```diff
@@ -0,0 +1,45 @@
+import Joi from '@hapi/joi';
```

Comment:
there is a file doing this for you bro. You just need to give it validation schemas. 
Because I know that it won't be easy to give custom messages for each and every field. I created a helper in constants for all of our validation schemas. I also created a middleware to validate them for you and give you custom messages if you want. Please, @alainburindi, kindly, check them out